### PR TITLE
Add kelvinlets entry

### LINKED ISSUE #45: Missing tutorial entry: 409_Kelvinlets
@SooryaN

### PR DESCRIPTION
**Pull request by SooryaN**
Fixes #45

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -1847,8 +1847,107 @@ bone and the final locations are stored in `U`.
...
+\mathbf{b} = 0$
+
+The Kelvinlet is the solution to the Navier-Cauchy equation in the case of a
+concentrated body load due to a force vector $\mathbf{f}$ at a point
+$\mathbf{x}_{0}$, i.e., where $\mathbf{b}(\mathbf{x}) = \mathbf{f}
+\delta(\mathbf{x} − \mathbf{x}_{0})$ and can be written as:
+
+$\mathbf{u}(\mathbf{r}) = \left[ \frac{(a - b)}{r}I +
+\frac{\mathbf{b}}{\mathit{r}^3}\mathbf{r}\mathbf{r}^{t}\right] \equiv
+\mathbf{K}(\mathbf{r})\mathbf{f}$
```
Comment (jdumas) [inline docs/tutorial.md:1884]: I think you do not explain what is `K`, and why it depends on `r` here.

File: docs/tutorial.md [hunk 2]
```diff
@@ -1847,8 +1847,107 @@ bone and the final locations are stored in `U`.
...
+\mathbf{K}(\mathbf{r_{\epsilon}})\mathbf{f}$
+
+where $\mathbf{r}_{\epsilon} = \sqrt{r^2 + \epsilon^2}$ is the regularized
+distance, $\epsilon > 0$ is the radius of the sculpting brush.
+
+Thus, given a force vector $\mathbf{f}$, the displacement for any point in $R^3$
+can be calculated, thus defining the physically based space deformation. In
+practice, the force vector is parameterized in terms of the brush tip
+displacement $\mathbf{\bar{u}}$ as $\mathbf{u_{\epsilon}} =
+c\epsilon\mathbf{K_{\epsilon}}(\mathbf{r})\mathbf{\bar{u}}$
```
Comment (jdumas) [inline docs/tutorial.md:1925]: What is `c` here?

**Review by jdumas — COMMENTED** [review]
Thanks for the writing this tutorial! I think there are two symbols that are missing a description, please let me know if I missed them.

File: docs/tutorial.md (hunk 2 above)
Comment (SooryaN) [reply docs/tutorial.md]: Ah, looks like I missed that. Added now

File: docs/tutorial.md (hunk 1 above)
Comment (SooryaN) [reply docs/tutorial.md:1884]: K is the kelvinlet function. Will make that explicit

**Comment by jdumas** [comment]
Thanks. I fixed the remaining typos. Merging now!

**Pull request merged**